library/ssl_tls.c

```diff
@@ -379,7 +379,7 @@ static int tls_prf_generic( mbedtls_md_type_t md_type,
 {
     size_t nb;
     size_t i, j, k, md_len;
-    unsigned char tmp[128];
+    unsigned char tmp[132];
```

Comment:
This change is done to pass the condition (and avoid overflow):
~~~
 if( sizeof( tmp ) < md_len + strlen( label ) + rlen )
        return( MBEDTLS_ERR_SSL_BAD_INPUT_DATA );
~~~
In this case, the sizeof `md_len + strlen( label ) + rlen )` is 131, as it's a long label than others.
How do you suggest to name the macro of this tmp buffer?
Perhaps it would be better to assign this buffer on the heap, to avoid future issues as well?